### DISCUSSION TITLE
Why osu use too much ram

### DISCUSSION THREAD

**Discussion by rXelelo** [ppy/osu#29655] [Q&A] [answered]
-->
Name|User|CPU|ID|RAM
![Знімок екрана з 2024-08-30 11-03-58](https://github.com/user-attachments/assets/190adb4b-56c2-4c39-b5a8-fe8c078cfc1d)

**Comment by peppy** [comment, marked as answer]
Dunno

**Reply by rXelelo** [reply]
wtf ppy

**Reply by peppy** [reply]
If you actually want to be helpful, you can provide logs, provide a memory dump from task manger, and explain how you got into this state.
